pbsmetrics/go_metrics.go

```diff
@@ -89,7 +89,7 @@ const unknownBidder openrtb_ext.BidderName = "unknown"
 // rather than loading legacy metrics that never get filled.
 // This will also eventually let us configure metrics, such as setting a limited set of metrics
 // for a production instance, and then expanding again when we need more debugging.
-func NewBlankMetrics(registry metrics.Registry, exchanges []openrtb_ext.BidderName) *Metrics {
+func NewBlankMetrics(registry metrics.Registry, exchanges []openrtb_ext.BidderName, disableAccountMetrics bool) *Metrics {
```

Comment:
I mentioned earlier that we should shove this boolean into its own struct/object. We should then pass this object in rather than just the boolean value, so if we expand the list, we won't have to change the function signature every time.

Reply:
You are absolutely right.